st2common/tests/unit/services/test_policy.py

```diff
@@ -30,11 +30,10 @@
 from st2common.services import policies as policy_service
 
 import st2tests
+from st2tests.fixtures.generic.fixture import PACK_NAME as PACK
```

Comment:
Many tests already used a constant to define the PACK name, so all I had to do was import the constant from the fixture instead of hard-coding it in the test files.